planning/planning_debug_tools/msg/TrajectoryDebugInfo.msg

```diff
@@ -3,4 +3,5 @@ uint32 size
 float64[] arclength
 float64[] curvature
 float64[] velocity
+float64[] acceleration
```

Comment:
@takayuki5168 
I suggest to separate info some message for purpose because plot juggler will drop if array is long because of buffer.
for example
```
builtin_interfaces/Time stamp
uint32 size
float64[] arclength
float64[] velocity
float64[] acceleration
```

```
builtin_interfaces/Time stamp
uint32 size
float64[] arclength
float64[] yaw
float64[] curvature
```

Reply:
Memo consider topic drop if it happends